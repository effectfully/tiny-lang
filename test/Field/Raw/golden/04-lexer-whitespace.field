
  assert T;
/**
  * Checking how whitespace at the begging of the file would behave
  */
