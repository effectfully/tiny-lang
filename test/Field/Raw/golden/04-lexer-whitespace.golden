Program (Statements [EAssert (EConst (Some (UniConst Bool True)))])